features/05_use_rspec_matchers/command/run_too_long.feature

```diff
@@ -25,3 +25,24 @@ Feature: Check if a timeout occured during command execution
     """
     When I run `rspec`
     Then the specs should all pass
+
```

Comment:
This should be on its own feature file. :astonished:.

https://github.com/cucumber/aruba/blob/master/features/05_use_rspec_matchers/command/have_finished_within_time.feature

Did you rebase your PR?